R/compat-plyr.R

```diff
@@ -374,13 +277,13 @@ dapply <- function(df, by, fun, ..., drop = TRUE) {
   apply_fun <- function(x) {
     res <- fun(x, ...)
     if (is.null(res)) return(res)
-    if (length(res) == 0) return(new_data_frame())
+    if (length(res) == 0) return(data_frame(.name_repair = "minimal"))
```

Comment:
Using `vctrs::new_data_frame()` is probably fine here if you wanted to

Reply:
What does `.name_repair = "minimal"` do with no inputs? It's the same as `data_frame()` right?

Reply:
@lionel- it is just comparatively very slow because of the name repair stuff

``` r
bench::mark(vctrs::data_frame(),vctrs::data_frame(.name_repair = "minimal"))
#> # A tibble: 2 × 6
#>   expression                                       min   median `itr/sec`
#>   <bch:expr>                                  <bch:tm> <bch:tm>     <dbl>
#> 1 vctrs::data_frame()                          20.64µs  22.15µs    40510.
#> 2 vctrs::data_frame(.name_repair = "minimal")   3.28µs   3.69µs   232047.
#> # … with 2 more variables: mem_alloc <bch:byt>, `gc/sec` <dbl>
```

<sup>Created on 2022-06-16 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>